frontend/src/plugin/registry.tsx

```diff
@@ -126,4 +128,16 @@ export default class Registry {
   registerDetailsViewSection(sectionName: string, sectionFunc: sectionFunc) {
     store.dispatch(setDetailsView(sectionName, sectionFunc));
   }
+  /**
+   *
+   * @param branding all the branding change related resources that are specified in the ```JSX BrandingProps``` interface
+   *
+   * @example
+   * ```JSX
+   * register.registerBrandingChange({smallScreenLogo: File, largeScreenLogo: File})
+   * ```
+   */
+  registerBrandingChange(branding: BrandingProps) {
```

Comment:
I thought I had left already a comment in a different review explaining that "branding" is a broad term, and that this plugin changes the app logo which should be specific name. Maybe `registerMainLogo`.